### PR TITLE
Accent

### PR DESCRIPTION
**Pull request by ptbrowne**
- [x] Created project in Accent
- [x] Imported translations to Accent
- [x] Added config and scripts to push / pull translations to Accent

### CODE REVIEW

**Comment by wereHamster** [comment]
Accent supports showing comments in the UI (only in gettext files though): https://github.com/mirego/accent/issues/135#issuecomment-1418452432

I was never able to test that, since all my project use JSON and not gettext files. It looks like `accent sync` does not include any comments in the .po files (judging by the diff):

image

**Comment by ptbrowne** [comment]
Thanks for this, I opened a new issue for Accent so that they preserve the comments when pulling. To circumvent this problem and preserve comments, I now re-run `lingui extract` after pulling the locales from accent. It seems it does the job well. Now the diff of this PR is much cleaner :)

File: README.md [hunk 1]
```diff
@@ -14,33 +14,35 @@
...
 
 A public documentation is available at https://visualize.admin.ch/docs/.
 
-## Development Environment
+## 2. <a name='DevelopmentEnvironment'></a>Development Environment
 
 To start the development environment, you need a Docker runtime, e.g. [Docker Desktop](https://www.docker.com/products/docker-desktop) and [Nix](https://nixos.org).
 
-### Setting up the dev environment
+### 2.1. <a name='Settingupthedevenvironment'></a>Setting up the dev environment
```
Comment (bprusinowski) [inline README.md:45]: Looks like all the titles were concatenated together? Why did you change to the `a` tag? 😄

File: package.json [hunk 2]
```diff
@@ -23,8 +23,11 @@
     "start": "yarn db:migrate && GLOBAL_AGENT_ENVIRONMENT_VARIABLE_NAMESPACE='' NO_PROXY='localhost,127.0.0.1' next start ./app -p $PORT",
     "typecheck": "lingui compile && tsc --noEmit -p ./app && tsc --noEmit -p ./embed",
     "lint": "eslint app -c .eslintrc.json",
-    "locales:extract": "NODE_ENV=development lingui extract --verbose; ./scripts/strip-locale-line-numbers.sh",
+    "locales:extract": "NODE_ENV=development lingui extract --clean; ./scripts/strip-locale-line-numbers.sh",
```
Comment (bprusinowski) [inline package.json:26]: 💯

**Review by bprusinowski — APPROVED** [review]
Nice :) Will the client have access to Accent, or is it for our internal purposes?

File: README.md (hunk 1 above)
Comment (ptbrowne) [reply README.md:45]: It was automatically done by the Markdown TOC plugin I have on vscode, when I updated the table of contents. The link is empty, it is so that the table of contents works.
